deploy/databases/docker-compose-macos.yaml

```diff
@@ -0,0 +1,43 @@
+version: "3.9"
+services:
+
+  # PostgreSQL
+  # ------------------------------
+  postgres:
+    image: postgres:16
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=${SDDB_USER}
+      - POSTGRES_PASSWORD=${SDDB_PASS}
+      - POSTGRES_DB=${SDDB_DATABASE}
+
+  # MySQL
+  # ------------------------------
+  mysql:
+    image: mysql:8.2
+    ports:
+      - '3306:3306'
+    environment:
+      MYSQL_USER: ${SDDB_USER}
+      MYSQL_PASSWORD: ${SDDB_PASS}
+      MYSQL_DATABASE: ${SDDB_DATABASE}
+      MYSQL_ROOT_PASSWORD: 'root'
+
+
+  # Microsoft SQL Server
+  # ------------------------------
+  mssql:
+    image: mcr.microsoft.com/azure-sql-edge
```

Comment:
The only difference I see from `docker-compose` is that you replace MSSQL with Azure-SQL.

What is the difference between them ? 

Is it the same thing ?
If yes, so should stick to one of them.
If no, it should go to another section.

Reply:
This one is compatible with macOS